test/gurka/test_elevation.cc

```diff
@@ -230,4 +237,28 @@ TEST(Standalone, ElevationCompareToSkadi) {
       EXPECT_FALSE(elevation && elevation->IsArray());
     }
   }
+
+  // Sanity check that conditional speed limits were not corrupted
+  auto route = gurka::do_action(valhalla::Options::route, map, {"S", "F"}, "auto");
+  const auto leg = route.trip().routes(0).legs(0);
+  ASSERT_EQ(leg.node_size(), 5); // 4 edges + last node
+  for (int i = 0; i < 4; ++i) {
+    ASSERT_EQ(leg.node(i).edge().conditional_speed_limits_size(), 1);
+    EXPECT_EQ(leg.node(i).edge().conditional_speed_limits(0).speed_limit(), 30);
+
+    // 19:00-06:00
+    const auto& condition = leg.node(i).edge().conditional_speed_limits(0).condition();
+    EXPECT_EQ(condition.day_dow_type(), TripLeg_TimeDomain_DayDowType_kDayOfMonth);
+    EXPECT_EQ(condition.dow_mask(), 0);
+    EXPECT_EQ(condition.begin_hrs(), 19);
+    EXPECT_EQ(condition.begin_mins(), 0);
+    EXPECT_EQ(condition.begin_month(), 0);
+    EXPECT_EQ(condition.begin_day_dow(), 0);
+    EXPECT_EQ(condition.begin_week(), 0);
+    EXPECT_EQ(condition.end_hrs(), 6);
+    EXPECT_EQ(condition.end_mins(), 0);
+    EXPECT_EQ(condition.end_month(), 0);
+    EXPECT_EQ(condition.end_day_dow(), 0);
+    EXPECT_EQ(condition.end_week(), 0);
+  }
```

Comment:
Why add this in the elevation test?

Reply:
This test helped me to find and fix a bug in the initial implementation. Removed in 58bf2016c51e512002253d39f0ac55a104296a23